### PR TITLE
[FSPE-4663] Added support of keyboard control to Tabs

### PR DESCRIPTION
**Pull request by janibolkvadze**
https://ctl.atlassian.net/browse/FSPE-4663

@dani-cl-madrid @SergioQCL @mattnickles @jllr 

https://ctl.atlassian.net/browse/FSPE-4663

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f61ce2088b8c98a1eb71e36f2a8eac33864d89d6-chi-pr.enterprise.ctl.io" target="_blank"> https://f61ce2088b8c98a1eb71e36f2a8eac33864d89d6-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1699/ for more details.

**Review by jllr — APPROVED** [review]

File: src/chi/javascript/components/tab.js [hunk 1]
```diff
@@ -39,8 +51,67 @@ class Tab extends Component {
...
+      (tab, index) => {
+        tab.addEventListener('keyup', (e) => this.keyupEventListener(e, index));
+      }
+    );
   }
 
+  keyupEventListener(event, index) {
+    const key = event.keyCode,
+      vertical = this._elem.classList.contains(CLASS_VERTICAL),
+      activeTabIndex = Array.from(this._tabs).indexOf(this.getActiveTab());
```
Comment (SergioQCL) [inline src/chi/javascript/components/tab.js:65]: Please, define one const per row.

**Review by SergioQCL — CHANGES_REQUESTED** [review]

File: src/chi/javascript/components/tab.js (hunk 1 above)
Comment (janibolkvadze) [reply src/chi/javascript/components/tab.js]: Done 😉

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://bf3c3f0e5736de076666ffa0cc3a77c43044e3ee-chi-pr.enterprise.ctl.io" target="_blank"> https://bf3c3f0e5736de076666ffa0cc3a77c43044e3ee-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1718/ for more details.

**Review by SergioQCL — APPROVED** [review]
ACK

**Review by mattnickles — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-lxkiwm, https://ops.elasticbox.com/services/instances/i-19qqyr

**Pull request merged**